openapi-v1.yaml

```diff
@@ -2041,6 +2041,15 @@ definitions:
           type: string
           x-omitempty: true
 
+  TileDBCloudUDFRuntimeProfile:
+    description: >
+      The runtime profile to use for the UDF execution. Profiles define resource limits for memory and cpus.
+      Check the tiledb cloud provider documentation for the details.
```

Comment:
`tiledb cloud` -> `TileDB Cloud`

Also it'd be great to provide a hyperlink but I understand such docs may currently be in flux ...